e2e-tests/gef/cypress/integration/mandatory-criteria.spec.js

```diff
@@ -0,0 +1,54 @@
+import relative from './relativeURL'
+
+context('Mandatory Criteria Page', () => {
+  before(() => {
+    cy.fixture('login')
+      .then((res) => {
+        cy.login(res.MAKER)
+      })
+    
+    cy.on('uncaught:exception', () => false)
+  })
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('connect.sid')
+    cy.visit(relative('/gef/mandatory-criteria'))
+  })
+
+  describe('Visiting page', () => {
+    it ('displays the header', () => {
+      cy.get('[data-cy="heading-caption"]')
+      cy.get('[data-cy="main-heading"]')
+    })
+
+    it ('displays the mandatory criteria text', () => {
+      cy.get('[data-cy="mandatory-criteria"]')
+    })
+  })
+
+  describe('Clicking on Continue', () => {
+    it ('returns error when no radio button has been selected', () => {
+      cy.get('[data-cy="form"]').submit()
+      cy.get('[data-cy="mandatory-criteria-error"]')
+    })
+
+    it('redirects the user to ** when they select False', () => {
+      cy.get('[data-cy="mandatory-criteria-false"]').click()
+      cy.get('[data-cy="form"]').submit()
+      cy.url().should('eq', relative('/gef/mandatory-criteria'))
+    })
+
+    it('redirects the user to the Name Application page when they select True', () => {
+      cy.get('[data-cy="mandatory-criteria-true"]').click()
+      cy.get('[data-cy="form"]').submit()
+      cy.url().should('eq', relative('/gef/name-application'))
+    })
+  })
+
+  describe('Clicking on Cancel', () => {
+    it('keeps the user on the same page FOR NOW', () => {
+      cy.get('[data-cy="cancel-button"]').click()
```

Comment:
would be nice to use a 'page' file that has all the selectors for a given page, but not a biggie. eg https://github.com/foundry4/dtfs2/blob/master/e2e-tests/portal/cypress/integration/pages/eligibilityCriteria.js

Reply:
Done